### PR TITLE
Allow for IPI clusters with FIPS enabled

### LINKED ISSUE #676: Error Installing a FIPS-Enabled IPI Cluster
While trying to provision an IPI cluster with FIPS, the following error is returned ([rehearsal job](https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_release/55472/rehearse-55472-periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.17-lp-interop-mtc-interop-aws-fips/1823355304834764800)):

```
level=error msg=failed to fetch Metadata: failed to load asset "Install Config": failed to create install config: invalid "install-config.yaml" file: fips: Forbidden: target cluster is in FIPS mode, use the FIPS-capable installer binary for RHEL 9 on a host with FIPS enabled.
level=error msg=To obtain a suitable binary, download the openshift-install-rhel9 archive from the client mirror, or extract the openshift-install-fips command from the release payload.
```

The cluster config(s) being used are as follows ([pull request](https://github.com/openshift/release/pull/55472)):
```yaml
CLUSTER1_CONFIG: name-prefix=mtc-source;base-domain=cspilp.interop.ccitredhat.com;platform=aws;region=us-east-2;version=4.17;stream=stable;worker-flavor=m5.xlarge;timeout=2h;fips=true
CLUSTER2_CONFIG: name-prefix=mtc-target;base-domain=cspilp.interop.ccitredhat.com;platform=aws;region=us-east-2;version=4.17;stream=nightly;worker-flavor=m5.xlarge;timeout=2h;fips=true
```

### PR DESCRIPTION
**Pull request by calebevans**
Closes #676

### CODE REVIEW

File: openshift_cli_installer/libs/clusters/ipi_cluster.py [hunk 1]
```diff
@@ -64,7 +66,7 @@ def _prepare_ipi_cluster(self) -> None:
             self._create_install_config_file()
 
     def _ipi_download_installer(self) -> None:
-        openshift_install_str = "openshift-install"
+        openshift_install_str = f"openshift-install{'-fips' if self.cluster_info['fips'] else ''}"
```
Comment (myakove) [inline openshift_cli_installer/libs/clusters/ipi_cluster.py:69]: `openshift_install_str` represents `openshift-install` as str.
either include your code when we call the command (line 79) or save it on a new content and use it in the command.

**Review by myakove — CHANGES_REQUESTED** [review]

File: openshift_cli_installer/libs/clusters/ipi_cluster.py (hunk 1 above)
Comment (rnetser) [reply openshift_cli_installer/libs/clusters/ipi_cluster.py:69]: or rename to `openshift_install_binary`

**Review by myakove — APPROVED** [review]

**Comment by msi-b1** [comment]
New container for quay.io/redhat_msi/openshift-cli-installer:latest published

**Pull request merged**